net/message/message.go

```diff
@@ -103,13 +103,13 @@ func (m *QueryResponse) Type() byte {
 
 // SendKeysMessage is sent by both parties to each other to initiate the protocol
 type SendKeysMessage struct {
-	OfferID            types.Hash              `json:"offerID"`
+	OfferID            types.Hash              `json:"offerID"` // Not set by XMR Maker
 	ProvidedAmount     *apd.Decimal            `json:"providedAmount" validate:"required"`
 	PublicSpendKey     *mcrypto.PublicKey      `json:"publicSpendKey" validate:"required"`
 	PrivateViewKey     *mcrypto.PrivateViewKey `json:"privateViewKey" validate:"required"`
-	DLEqProof          string                  `json:"dleqProof" validate:"required"`
+	DLEqProof          []byte                  `json:"dleqProof" validate:"required"`
```

Comment:
This is what we discussed before. The proof is fairly large and sending it in base64 (automatic when using a byte slice) instead of base16, reduces the size by 4x.